spec/manageiq_performance/configuration_spec.rb

```diff
@@ -84,92 +157,7 @@
   end
 
   describe "default config" do
-    it "defines ManageIQPerformance.config.default_dir" do
-      expect(ManageIQPerformance.config.default_dir).to eq("tmp/manageiq_performance")
-      expect(ManageIQPerformance.config["default_dir"]).to eq("tmp/manageiq_performance")
```

Comment:
I know this stuff is verbose, but it is very easy to read / understand.

(kinda like the booring way of doing this)